exporter/containerimage/writer.go

```diff
@@ -212,7 +212,22 @@ func (ic *ImageWriter) exportLayers(ctx context.Context, refCfg cacheconfig.RefC
 	return out, err
 }
 
-func (ic *ImageWriter) commitDistributionManifest(ctx context.Context, ref cache.ImmutableRef, config []byte, remote *solver.Remote, oci bool, inlineCache []byte, buildInfo []byte) (*ocispecs.Descriptor, *ocispecs.Descriptor, error) {
+func appendExtraLayer(ctx context.Context, ref cache.ImmutableRef, refCfg cacheconfig.RefConfig, descs []ocispecs.Descriptor, sg session.Group) ([]ocispecs.Descriptor, error) {
+	if refCfg.Compression.Type == compression.Nydus {
+		// For nydus format, appending an extra nydus bootstrap layer to the manifest,
+		// this layer represents the whole metadata of filesystem view for the entire image.
+		// The nydus merge operation is fast, so we don't need to create a new cache
+		// record for this extra layer, the layer will be gc after lease expired.
+		desc, err := cache.MergeNydus(ctx, ref, refCfg.Compression, sg)
```

Comment:
Is the same exception needed for all other exporter as well? I think this should be handled by the cache.

Reply:
Thanks for the reply. Nydus adds an extra bootstrap layer mainly for the performance optimization of the image scenario, so there is no need to export this extra layer for other exporters. If it is handled in the cache, does an extra cache record need to be added? I am worried that it will affect the processing of other parts after `GetRemotes`.